docs/hello_nextflow/10_hello_seqera.md

```diff
@@ -1,7 +1,188 @@
+---
+title: Hello Seqera
+description: Get started with Seqera Platform
+---
+
 # Part 9: Hello Seqera
 
-TODO
+So far we've been running Nextflow workflows on our local machine using the command line interface.
+In this section, we'll introduce you to Seqera Platform, a powerful cloud-based platform for running, monitoring, and sharing Nextflow workflows.
 
----
+!!! tip
+
+    [Sign up](https://cloud.seqera.io/) to try Seqera for free or request a [demo](https://seqera.io/demo/) for deployments in your own on-premise or cloud environment.
+
+You'll learn how to use Seqera Platform within Nextflow CLI, the Seqera Platform GUI, and the API.
```

Comment:
```suggestion
You'll learn how to use Seqera Platform from the Nextflow CLI, the Seqera Platform GUI, and the API.
```

Maybe expand out CLI, GUI, API etc first time you use them? Not sure everyone will be nerdy enough to get them by default.